docs/config.md

```diff
@@ -110,6 +110,11 @@ Additional scrapes can be added through files prefixed with `scrape_`. By defaul
 The simplest way to add scrape configs is to use Docker [secrets](https://docs.docker.com/engine/swarm/secrets/) or [configs](https://docs.docker.com/engine/swarm/configs/).
 
 
-## Scrape Label Configuration
+## Scrape Label Configuration With Service and Node Labels
 
-When using a version of [Docker Flow Swarm Listener](https://github.com/vfarcic/docker-flow-swarm-listener), DFSL, newer than `18.02.06-31`, you can configure DFSL to send node node hostnames to `Docker Flow Monitor`, DFM. This can be done by setting `DF_INCLUDE_NODE_IP_INFO` to `true` in the DFSL environment. DFM will automatically display the node hostnames as a label for each prometheus target. The `DF_SCRAPE_TARGET_LABELS` env variable allows for additional labels to be displayed. For example, if a service has env variables `com.df.env=prod` and `com.df.domain=frontend`, you can set `DF_SCRAPE_TARGET_LABELS=env,domain` in DFM to display the `prod` and `frontend` labels in prometheus.
+When using a version of [Docker Flow Swarm Listener](https://github.com/vfarcic/docker-flow-swarm-listener), DFSL, newer than `18.03.20-39`, you can configure DFSL to send node information to `Docker Flow Monitor`, DFM. This can be done by setting `DF_INCLUDE_NODE_IP_INFO` to `true` in the DFSL environment. DFM will automatically display the node hostnames as a label for each prometheus target. The `DF_SCRAPE_TARGET_LABELS` env variable allows for additional labels to be displayed. For example, if a service has env variables `com.df.env=prod` and `com.df.domain=frontend`, you can set `DF_SCRAPE_TARGET_LABELS=env,domain` in DFM to display the `prod` and `frontend` labels in prometheus.
+
+In addition to service labels, DFM can be configured to import node and engine labels prefixed with `com.df.` as prometheus labels for our targets. First, configure DFSL to push node events to DFM by setting `DF_NOTIFY_CREATE_NODE_URL=[MONITOR_IP]:[MONITOR_PORT]/v1/docker-flow-monitor/node/reconfigure` and `DF_NOTIFY_REMOVE_NODE_URL=[MONITOR_IP]:[MONITOR_PORT]/v1/docker-flow-monitor/node/remove` in DFSL. Then, in DFM, set `DF_GET_NODES_URL=[SWARM_LISTENER_IP]:[SWARM_LISTENER_PORT]/v1/docker-flow-swarm-listener/get-nodes` and set `DF_NODE_TARGET_LABELS` env variable to a comma seperate list of labels use. For example, if our node has label `com.df.aws-region=us-east-1` and we set `DF_NODE_TARGET_LABELS=aws-region`, your prometheus targets on that node will include `aws-region=us-east-1`. For more information, please visit the [Flexiable Labeling tutorial](tutorial-flexible-labeling.md).
```

Comment:
Maybe add a reference to a YAML or some other path of the tutorial that has an example with `DF_NOTIFY_CREATE_NODE_URL`?